reading-notes-intro-computation-programming-python/chapter-one_getting-started/quiz.md

```diff
@@ -3,19 +3,28 @@
 ## What is a computer ?
 
 What does `computer` mean ? 
+> A machine that performs calculations and remembers the results of the calculations, very quickly. Those calculations are executed following a program, a finite sequence or list of instructions.
+
 You can use Wikipedia to find out.
 Find also a usage related to Easter. 
+> Something related to Blaise Pascal, so I would say the Pascaline, the first known calculator.
 
-Is there such things as a human computer ?
+Is there such a thing as a human computer ?
+> Indeed, according to wikipedia and etymology of the word computer, since early-17th century the term is "_referred to a human computer, a person who carried out calculations or computations_".
 
 Why do we name computers "computers" ?
+> Because they perform calculation of mathematical or logical operations also called computations.
```

Comment:
As in [Wikipedia](https://en.wikipedia.org/wiki/Computer_(disambiguation)) disambiguation page, I wanted you to say we use "computer" when speaking about machine, because most of computer are nowadays machines.